Framework/PythonInterface/plugins/functions/FMuF.py

```diff
@@ -0,0 +1,38 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+# pylint: disable=invalid-name, anomalous-backslash-in-string, attribute-defined-outside-init
+
+from mantid.api import IFunction1D, FunctionFactory
+import numpy as np
+
+
+class FmuF(IFunction1D):
+
+    def category(self):
+        return "Muon"
+
+    def init(self):
+        self.declareParameter("A0", 0.5, 'Amplitude')
+        self.declareParameter("FreqD", 0.2, 'Dipolar interaction frequency (MHz)' )
+        self.declareParameter("Lambda", 0.1, 'Exponential decay rate (MHz)')
+        self.declareParameter("Sigma", 0.2, 'Gaussian decay rate (MHz)')
```

Comment:
Are the units here correct? Or should they be removed as well?

Reply:
they should be removed. I dont think Hz is the appropriate unit for decay rate.